### PR TITLE
Add script to register Eservice with ledger

### PR DESCRIPTION
**Pull request by aaadesok**
Signed-off-by: Abdulkareem Adesokan <abdulkareem.adesokan@intel.com>

### CODE REVIEW

File: eservice/register_with_ledger.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Registers MR_ENCLAVE & BASENAMES with Ledger
+Register(){
+	echo "Register with ledger"
+	if [ ! -f $eservice_enclave_info_file ]; then
+		echo "Registration failed! eservice_enclave_info_file not found!"
+	else
+		cmd=`echo "../sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY --url $LEDGER_URL pdo.test.registry.measurements \`cat $eservice_enclave_info_file | grep -o 'MRENCLAVE:.*' | cut -f2- -d:\`"`
+		eval $cmd
+		cmd=`echo "../sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY --url $LEDGER_URL pdo.test.registry.measurements \`cat $eservice_enclave_info_file | grep -o 'BASENAME:.*' | cut -f2- -d:\`"`
```
Comment (bvavala) [inline eservice/register_with_ledger.sh]: L35: the settings name is wrong.
See https://github.com/hyperledger-labs/private-data-objects/blob/master/python/sawtooth/helpers/pdo_address_helper.py#L105-L112

File: eservice/register_with_ledger.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		eval $cmd
+	fi
+}
+
```
Comment (bvavala) [inline eservice/register_with_ledger.sh]: The registration of the verification key is missing.

File: eservice/register_with_ledger.sh (hunk 1 above)
Comment (aaadesok) [reply eservice/register_with_ledger.sh]: Fixed. 
PR will be merged with  - [PR#82](https://github.com/hyperledger-labs/private-data-objects/pull/82)

File: eservice/register_with_ledger.sh (hunk 2 above)
Comment (aaadesok) [reply eservice/register_with_ledger.sh]: Fixed. 
PR will be merged with  - [PR#82](https://github.com/hyperledger-labs/private-data-objects/pull/82)

**Pull request closed without merging**